Intersect.Client/Interface/Game/EntityPanel/EntityBox.cs

```diff
@@ -88,19 +90,32 @@ public partial class EntityBox
 
         public string[] PaperdollTextures;
 
-        public Button PartyLabel;
+        //public Button PartyLabel;
 
         public bool PlayerBox;
 
         public ImagePanel ShieldBar;
 
-        public Button TradeLabel;
+        //public Button TradeLabel;
 
         public bool UpdateStatuses;
```

Comment:
lets rename to ShouldUpdateStatuses, makes more sense in the following changes

Reply:
also updated PlayerStatusWindow's